src/services/FsData.service.ts

```diff
@@ -119,14 +101,24 @@ class FsData implements ng.IServiceProvider {
       datasourceIds.push(0);
     }
 
-    let requestUrl = `${this._baseUrl}${this._dataUrl}`.replace(/{{datasources}}/g, datasourceIds.join(',')).replace(/{{type}}/g, dataset);
+    var dataUrl = this._baseUrl + this._dataUrl;
+    var config = {
```

Comment:
We can pass a config object which will set the params without the need for curly braces